projects/plugins/jetpack/_inc/client/my-plan/my-plan-body.jsx

```diff
@@ -478,7 +478,7 @@ class MyPlanBody extends React.Component {
 											src={ imagePath + '/jetpack-google-analytics.svg' }
 											className="jp-landing__plan-features-icon"
 											alt={ __(
-												'Site stats showing an evolution in traffic and engagement',
+												'Jetpack Stats showing an evolution in traffic and engagement',
```

Comment:
I'm not sure if this is the correct replacement. This block corresponds to Google Analytics and this is the `alt` text description for the block's image.

This block is visible after adding `Jetpack Security Daily` plan - required in an attempt to configure Google Analytics from [Tools/Marketing menu](https://wordpress.com/marketing/traffic/).

I would suggest making this text more generic with `Stats showing an evolution in traffic and engagement` or more specific with `Google Analytics Stats showing an evolution in traffic and engagement`.

Reply:
Fixed in 8c67580.